Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -91,6 +93,19 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
         let url = URL(string: user.avatar.medium)
         return (url, canPostComment, false, inputAreaBecomeFirstResponder)
```

Comment:
Note: A lot of this will be subject to re-factor in @justinswart next PR.

Reply:
I don't think this will change will it? It's just the config for the composer view unless I'm not following something..

Reply:
Yeah, this comment is unclear. I meant we are updating the data source to handle more of the work and I'm leaving a comment here for historical context.